src/main/java/edu/greenblitz/tobyDetermined/Robot.java

```diff
@@ -65,6 +65,9 @@ private static void initSubsystems() {
 		IntakeExtender.init();
 		IntakeRoller.init();
 		OI.init();
+
+//		LED.getInstance(); //todo might make error
```

Comment:
Try not to leave commented out code and more importantly, why can LED.getInstance() throw an error? Sounds like a major bug to me